internal/neofs/neofs.go

```diff
@@ -246,6 +247,37 @@ func (x *NeoFS) DeleteContainer(ctx context.Context, id cid.ID, token *session.C
 	return nil
 }
 
+func (x *NeoFS) signMultipartObject(header *object.Object, signer neofscrypto.Signer) error {
+	var (
+		payloadChecksum = sha256.New()
+		sig             neofscrypto.Signature
+	)
+
+	payloadChecksum.Write(header.Payload())
+	header.SetPayloadChecksum(checksum.NewFromHash(checksum.SHA256, payloadChecksum))
```

Comment:
I thought you have them somewhere already for multiparts. Or you can calculate them on the go, just like regular ones. The rationale is that you'd have warmer caches then.

Reply:
It exists, but it calculates on the fly during payload writing to the network. Updated to recalculated hash

Reply:
But tz is still the same.

Reply:
Yes, we calculate `tz` for the whole Big object. Separately for each part, we didn't calculate it before. The node did it on my behalf. Now we calculate it in place